CMakeLists.txt

```diff
@@ -544,10 +544,7 @@ IF (CXXTEST_FOUND)
 		)
 	ENDIF (CMAKE_BUILD_TYPE STREQUAL "Coverage")
 	ADD_CUSTOM_TARGET(test_python
-		DEPENDS PythonEvalUTest
-				PythonUtilitiesUTest
-				bindlink_cython atomspace_cython
-				scheme_wrapper logger_cython utilities_cython
+		DEPENDS tests PythonEval logger_cython scheme_wrapper bindlink_cython
```

Comment:
This seems to be a misunderstanding. The original intent was to be able to say `make test_python` and as a result, ONLY the python-related tests would run. This can save a lot of time during development, by avoiding the overhead of re-running all of the other tests.   This change seems to completely wreck the intended behavior.

Reply:
Just to be clear: if one says `make test`, then all of the python tests should *also* run -- The `make test_python` is just a short-cut, to run a subset of all of the unit tests.

Reply:
Thanks Linas, there were some misunderstanding on my side, now make test should successfully run python tests, i updated dependencies